imod/prepare/wells.py

```diff
@@ -25,21 +25,42 @@ def vectorized_overlap(bounds_a, bounds_b):
     )
 
 
+def compute_point_filter_overlap(bounds_wells, bounds_layers):
+    """
+    Special case for filters with zero filter length, these are set to layer
+    thickness.
+    """
+    zero_filter_length = bounds_wells[:, 1] == bounds_wells[:, 0]
+    in_layer = (bounds_layers[:, 1] >= bounds_wells[:, 1]) & (
+        bounds_layers[:, 0] < bounds_wells[:, 0]
+    )
+    layer_thickness = bounds_layers[:, 1] - bounds_layers[:, 0]
+    return zero_filter_length.astype(float) * in_layer.astype(float) * layer_thickness
```

Comment:
Could the following snippet also work?
`zero_filter_lengths = layer_thickness[zero_filter_length & in_layer]`